src/winetricks

```diff
@@ -2341,6 +2344,28 @@ _EOF_
 
 #---- Private Functions ----
 
+winetricks_dl_warning() {
+    case $LANG in
+    ru*) _W_countrymsg="Скрипт определил, что ваш IP адрес принадлежит России. Если во время загрузки файлов вы увидите ошибки несоответствия сертификата, скачайте файлы вручную или используйте VPN."
+      ;;
+    *)  _W_countrymsg="Your IP address has been determined to belong to Russia. If you encounter a certificate error while downloading, please use VPN or download files manually."
+      ;;
+    esac
+
+    # Lookup own country via IP address only once (i.e. don't run this function for every download invocation)
+    if [ -z "$MYCOUNTRY" -a -x "`which geoiplookup 2>/dev/null`" ]
+    then
+        ip_address=`wget http://ipinfo.io/ip -q -O -`
+        export MYCOUNTRY=`geoiplookup "$ip_address" | awk '/Country Edition/{print $4}' | sed 's/,//'`
```

Comment:
Rather than depending on geoiplookup, you could use http://ipinfo.io/..

ip="$(wget http://ipinfo.io/ip -q -O -)"
country="$(wget -q -O - http://ipinfo.io/${ip} | grep country | cut -d: -f2| tr -d "," | tr -d \" )"